pykwasm/src/pykwasm/kdist/wasm-semantics/ulm-wasm.md

```diff
@@ -180,8 +180,8 @@ In the standalone semantics, the Wasm VM obtains an entrypoint from the configur
 
 ```local
     syntax WasmString ::= #getEntryPoint() [function, total]
-    rule #getEntryPoint() => FUNCNAME
-         [[ <entry> FUNCNAME </entry> ]]
+    rule [[ #getEntryPoint() => FUNCNAME ]]
```

Comment:
Wow. I noticed this failed to compile and I kept trying different permutations of config cells, moving the configuration declaration around, etc... trying to get it to run. Somehow I forgot that the rule is bracketed and not the configuration cells. 😵

Reply:
well, it was actually @virgil-serbanuta who knew the right incantation :-)